### ISSUE TITLE
No Factions from Operation Trebuchet display in Zeus

### ISSUE THREAD

**Issue by Sully995** [zen-mod/ZEN#659]
**Arma 3 Version:** `2.06.148470`
**CBA Version:** `v3.15.6.211004`
**ZEN Version:** `1.12.1` 

**Mods:**
```
- CBA_A3
- ZEN
- Operation Trebuchet
```

**Description:**
Upon opening the Zeus interface to begin spawning units and/or vehicles. You can see that the UNSCDF and Insurrectionists headers for the types of units. Do not appear in the Zeus Interface despite, said mods being loaded, admin privileges, all settings with the game master module, being on local hosted, or the dedicated server. We can confirm confidence in settings because, viewing the EMPTY tab, I can see "OPTRE Buildings", "OPTRE Corvette Objects", "OPTRE Frigate", and "OPTRE Objects." So its only having issues with the factions and vehicles. All options for ZEN do appear in Zeus interface, assets pre-placed within the editor will still appear in game. They can still be copied and pasted while in Zeus. 

**Expected behavior:**
For the Zeus interface to display the factions from OPTRE in the unit tree like it does with vanilla and countless other modded factions that our group has used recently and in the past with no/minimum issues.

**Steps to reproduce:**
Add the steps needed to reproduce the issue.

1. Open EDEN Editor.
2. Spawn playable unit.
3. Place Gamemaster module.
4. Configure gamemaster module for all addons or all + unofficial addons. With admin login access.
5. Save and copy to dedicated server.
6. Open in preview, local, and/or dedicated server.
7. Open Zeus interface.
8. Notice that the "UNSCDF" is missing but "NATO", "FIA", "CTRG" and all vanilla factions are all there.

**Where did the issue occur?**
- This issue occurred with preview, local host, and dedicated server.

**Additional information:**
We've used OPTRE + ZEN back in December of 2020 without this issue. With lots of other mods, including ACE3, TFAR, and LAMBS AI. However for testing I did as asked by one of the authors in the steam workshop page. I only loaded CBA_A3, OPTRE, and ZEN for troubleshooting for this report.

**RPT log file:**
- +(https://github.com/Sully995/Sully995-s-Log-Files/blob/main/Arma3_x64_2022-01-10_17-21-24.rpt)
**Screenshots:**

![IMG_0579](https://user-images.githubusercontent.com/97484537/148859628-4fe80830-9efd-4800-b77c-ce84ec18ccca.jpg)
Only CBA, ZEN and OPTRE are loaded in.
![IMG_0580](https://user-images.githubusercontent.com/97484537/148859650-690475ba-9b36-49e2-86d0-3de4b85a4b16.jpg)
Notice the missing UNSCDF faction.
![IMG_0581](https://user-images.githubusercontent.com/97484537/148859659-de72c548-0667-4a86-a845-d0060afc1da3.jpg)
Zeus Interface will still display all the options for static objects from the mod.
![IMG_0582](https://user-images.githubusercontent.com/97484537/148859664-cff92546-0977-4070-9505-4b8133ab2224.jpg)
All features of ZEN works fine.

**Comment by mharis001** [comment]
Can you double check the faction filter settings?

> Configure gamemaster module for all addons or all + unofficial addons. With admin login access.

Also, confirm the issue persists with all addons (including unofficial ones) selected in the Game Master module.

**Comment by Kexanone** [comment]
Besides ZEN's faction filter, it could also be completely unrelated to ZEN. Did you check if the factions even appear if you run without ZEN?

**Comment by Sully995** [comment]
Loading without ZEN showed the faction trees. That gave me an idea though. I reloaded ZEN back in with CBA and OPTRE. I created a new player profile in the Arma 3 main menu. I then loaded up the same test mission and behold. The UNSCDF and Insurrectionists faction trees were visable and I'm now able to spawn OPTRE factions via Zeus. That has solved the problem for me. 

![IMG_0584](https://user-images.githubusercontent.com/97484537/149019754-8a418153-6fd3-4ffb-baac-fe780e5c29ec.jpg)
![IMG_0585](https://user-images.githubusercontent.com/97484537/149019756-23b0a242-1a7e-4e10-84d9-434cf8420a00.jpg)
